update the proposal process (#41)
swift-server/sswg: process/incubation.md

### Proposal

For a pitch to be moved into the Proposal stage, it must be endorsed by at least two members of the SSWG. The scope of the proposed code needs to closely align with the endorsed Pitch and it is subject to review based on the SSWG graduation criteria defined below. 

> Proposal are submitted to the SSWG by creating a PR in the [proposal area](https://github.com/swift-server/sswg/tree/master/proposals). Proposals follow [a template](https://github.com/swift-server/sswg/blob/master/proposals/0000-template.md) and include the following information:

Reviewer: ```suggestion
Proposals are submitted to the SSWG by creating a PR that adds the proposal document to the [proposal directory](https://github.com/swift-server/sswg/tree/master/proposals). Proposals follow [a template](https://github.com/swift-server/sswg/blob/master/proposals/0000-template.md) and include the following information:
```

Proposal are submitted to the SSWG by creating a PR in the [proposal area](https://github.com/swift-server/sswg/tree/master/proposals). Proposals follow [a template](https://github.com/swift-server/sswg/blob/master/proposals/0000-template.md) and include the following information:

* Name (must be unique within SSWG)
* Description (what it does, why it is valuable, origin, and history)
* Statement on alignment with SSWG mission
* Preferred initial maturity level (see SSWG Graduation Criteria)

> * Sponsor (where applicable)

Reviewer: This information should be removed from the template as well

* Social media accounts (optional)
* Community size and any existing sponsorship

> Once a propsal PR is submitted, the SSWG will assign a review manager during it's bi-weekely meeting. The review manager responsibilities include:
> 
> * Review the PR 
>   * Vaidate structure and language.
>   * Make sure the implementation is in place.
> * Update the PR
>   * Assign a number.
>   * Assign a review manager.
>   * Set status to "Active Review + Date range for review"
> * Secure approval from the PR author to merge changes above and merge them.
> * Publisha a form post in the [server proposals area](https://forums.swift.org/c/server/proposals) soliciting feedback from the community.
> * Watch the forum thread for conduct or off-topic issues, and make sure author is engaged.
> * Once the review period is over, summarize the key takeaways to the SSWG in writing.

Reviewer: ```suggestion
Once a Proposal PR is submitted, the SSWG will assign a review manager during it's bi-weekly meeting.

The review manager responsibilities include:
* Reviewing the PR:
  * Validating structure and language
  * Ensuring the implementation is "complete", as defined by the Pitch phase
* Updating the PR:
  * Assigning a Proposal number
  * Assigning the review manager
  * Updating the status to "Active Review (Date range for review)"
* Coordinating with the PR author to merge the updated PR details into the Proposal document
* Publishing a form post in the [server proposals area](https://forums.swift.org/c/server/proposals) to request implementation feedback from the community
* Moderating the forum thread for conduct and off-topic discussions, ensuring the author engages in discussion
* Summarizing key takeaways to the SSWG in writing, after the review has concluded
```

Author: we lost L55 and L56, intentional?

Reviewer: No, not intentional. Still getting used to GitHub's multi-line suggestions and those lines were lost.

Updated my suggestion to include those lines

The SSWG votes on pending proposals on a bi-weekly cadence, with the goal of voting on at least two proposals per month.

After the vote, the review manager will:

> * Annoucnce the vote results in the review thread.
> * Update the propsal's status based on the vote result.
> * Close ithe review thread.

Reviewer: ```suggestion
1. Announce the vote results in the review thread.
1. Update the Proposal's status based on the vote result.
1. Close the review thread.
```

Author: languages changes seem fine but we lost the header in L60 and not sure about `*` to `1.`

Reviewer: Line 60 should still include `After the vote, the review manager will:`?

GitHub shows that my suggestion only applies to Lines 61-63, L60 will remain as-is in the tree.

Regarding the switch to numbered list, this is mostly a technical writing nitpick. As the header implies a discrete list of steps the review manager will take, they should be numbered in the order. Above where we outline responsibilities, they're not in any particular order aside from what you (the author) decided made the most hierarchical sense.
